bridge/cypress/integration/uniform-integrations.spec.ts

```diff
@@ -425,6 +426,27 @@ describe('Add webhook subscriptions', () => {
   });
 });
 
+describe('Add secret from webhook subscriptions', () => {
```

Comment:
The implementation (should) also affects the edit page. Could you also add UI tests for the edit page?